{"owner": {"display_name": "Vinit", "reputation": 11, "profile_image": "https://www.gravatar.com/avatar/dfcabd8ca5bb23bab506d42398454646?s=128&d=identicon&r=PG&f=1", "user_id": 7084891, "user_type": "registered", "link": "http://stackoverflow.com/users/7084891/vinit"}, "link": "http://stackoverflow.com/questions/40304407/python-function-returning-22-invalid-argument-when-calling-from-jenkins-fra", "tags": ["python", "shell", "jenkins"], "creation_date": 1477655213, "score": 2, "last_edit_date": 1477655724, "is_answered": false, "view_count": 18, "last_activity_date": 1477655724, "question_id": 40304407, "answer_count": 0, "title": "python function returning &quot;22, &#39;Invalid argument&#39;&quot; when calling from jenkins framework"}